src/Domain/SalesforceWriteProxyRepository.php

```diff
@@ -65,6 +65,14 @@ public function pushAllPending(int $limit = 200): int
             ['id' => 'ASC'],
             $limit,
         );
+        
+        if (getenv('APP_ENV') === 'regression') {
+            $this->logger->info(sprintf(
+                'Donations to update: ',
+                $proxiesToUpdate,
```

Comment:
I'm not sure this is going to tell you much @tbg-jasonfung, have you tried it on local? I would expect you'll get a summary of the objects that shows they exist but not any of their properties.